server/odc-common/src/main/java/com/oceanbase/odc/common/util/SystemUtils.java

```diff
@@ -146,13 +151,103 @@ public static long getProcessPid(Process process) {
                 f.setAccessible(true);
                 pid = f.getLong(process);
                 f.setAccessible(false);
+            } else if (isOnWindows()) {
+                Field f = process.getClass().getDeclaredField("handle");
+                f.setAccessible(true);
+                pid = Kernel32.INSTANCE.getProcessId((Long) f.get(process));
+            } else {
+                throw new UnsupportedOperationException("Unsupported process class: " + process.getClass().getName());
             }
         } catch (Exception e) {
+            log.warn("get process id failed.", e);
             pid = -1;
         }
         return pid;
     }
 
+    public static boolean killProcessByPid(long pid) {
+        if (-1 == pid) {
+            throw new IllegalArgumentException("kill process by illegal argument pid: " + pid);
+        }
+        String command = "";
+        if (isOnWindows()) {
+            command = "cmd.exe /c taskkill /PID " + pid + " /F /T ";
+        } else {
+            command = "kill -9 " + pid;
+        }
+
+        return executeCommand(command, reader -> {
+            boolean result = false;
+            try {
+                String line;
+                while ((line = reader.readLine()) != null) {
+                    log.info(line);
+                }
+                result = true;
+            } catch (IOException e) {
+                log.warn("Reader from process output failed.", e);
+            }
+            return result;
+        });
+    }
+
+    public static boolean isProcessRunning(long pid) {
+        if (-1 == pid) {
+            throw new IllegalArgumentException("query process by illegal argument pid: " + pid);
+        }
+        String command = "";
+        if (isOnWindows()) {
+            // tasklist exit code is always 0. Parse output
+            // findstr exit code 0 if found pid, 1 if it doesn't
+            command = "cmd.exe /c \"tasklist /FI \"PID eq " + pid + "\" | findstr " + pid + "\"";
```

Comment:
There seems to be a problem with the splicing of this command, please confirm it.

Reply:
looks normally,  find process which pid equal 1 should like: cmd.exe /c \"tasklist /FI \"PID eq  1 \" | findstr 1 + \"